### ISSUE TITLE
Support for `gasLimit` as bigint seems to be broken

### ISSUE THREAD

**Issue by fmrsabino** [gelatodigital/relay-sdk#29]
With the introduction of [v5.0.0](https://github.com/gelatodigital/relay-sdk/releases/tag/v5.0.0) we saw the introduction of the support for `bigint`.

`chainId` seems to be serialised correctly: https://github.com/gelatodigital/relay-sdk/blob/db2718179439ecb4b5574ce5ecce516d5f28abfb/src/lib/sponsoredCall/index.ts#L65

However if we use `RelayRequestOptions.gasLimit` as a `bigint` we get the following error from the SDK:

```
GelatoRelaySDK/sponsoredCall: Failed with error: Do not know how to serialize a BigInt
```

It seems that the options are given "as-is" i.e.: no mapping involved.

https://github.com/gelatodigital/relay-sdk/blob/db2718179439ecb4b5574ce5ecce516d5f28abfb/src/lib/sponsoredCall/index.ts#L63

From my understanding, `bigint` serialisation is not done out of the box with `axios` so I assume that it comes from serialising the `POST` request with axios and we should map it to `string` beforehand.

**Comment by denis-pingin** [comment]
Thanks for reporting @fmrsabino, it's fixed in [v5.3.0](https://github.com/gelatodigital/relay-sdk/releases/tag/v5.3.0)

**Comment by fmrsabino** [comment]
> Thanks for reporting @fmrsabino, it's fixed in [v5.3.0](https://github.com/gelatodigital/relay-sdk/releases/tag/v5.3.0)

Thank you for the update @denis-pingin! Will test it with the new version.
